hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassActionsServiceImpl.java

```diff
@@ -102,6 +107,10 @@ public void setExecutor(ManagedExecutorService executor) {
         this.executor = executor;
     }
 
+    public void setSession(Session session) {
```

Comment:
It seems a little odd to Inject session and also have a setter for session, is that intended?